README.md

```diff
@@ -25,13 +25,52 @@ You can now access the api at [http://localhost:8000/api/v1/](http://localhost:8
 
 ### Usage
 
+#### Upload and validation of templates
+
 Upload templates using the following:
 
 ```bash
 curl --form template=@docx-template.docx --form name="Test Template" --form engine=docx-template http://localhost:8000/api/v1/template/
 ```
 
-And merge template with:
+**Hint:**
+
+You can enable additional validation of the template against a list of available
+placeholders by filling the `available_placeholders` attribute:
+
+```json
```

Comment:
So this means you can configure DMS to only allow the upload of templates with known placeholders?
If so, how do you configure it? Is there a curl command which could be documented?

Reply:
It's not config - it's part of the upload request - see #239

Reply:
I see :+1:. I was a bit confused by the mix of form data and JSON in the same request.